org.gnome.gThumb.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.gnome.gThumb
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+
+command: gthumb
+copy-icon: gthumb
```

Comment:
this has to be set as true, it's a boolean but I believe from the docs that it only works if you're also renaming the icons so you should have
>rename-icon: gthumb
>copy-icon: true